src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,18 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
+
+- For ASP.NET Core 6, .NET 6
+- For ASP.NET Core 5, .NET 5
+- For ASP.NET Core 3, .NET Core 3.1
+- For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
+
+Useful Links
+- [https://github.com/NLog/NLog.Web/releases](https://github.com/NLog/NLog.Web/releases)
```

Comment:
Not sure how it works, but would it not be nicer with pretty-url-names ? Think it is easier to understand `Full changelog` than the raw url `https://github.com/NLog/NLog.Web/releases`

Reply:
I agree, I will adjust those accordingly.  I think the [] part if the pretty name, and the () part is the link.  I have done it before, should not take long.

Reply:
changed